cmd/maya-apiserver/app/server/backup_endpoint_v1alpha1.go

```diff
@@ -409,3 +414,113 @@ func updateBackupStatus(clientset versioned.Interface, bkp *v1alpha1.CStorBackup
 		return
 	}
 }
+
+// delete is http handler which handles backup delete request
+func (bOps *backupAPIOps) delete() (interface{}, error) {
+	// Extract name of backup from path after trimming
+	backup := strings.TrimSpace(strings.TrimPrefix(bOps.req.URL.Path, "/latest/backups/"))
+
+	// volname is the volume name in the query params
+	volname := bOps.req.URL.Query().Get("volume")
+
+	// namespace is the namespace(pvc namespace) name in the query params
+	namespace := bOps.req.URL.Query().Get("namespace")
+
+	// schedule name is the schedule name for the given backup, for non-scheduled backup it will be backup name
+	scheduleName := bOps.req.URL.Query().Get("schedule")
+
+	if len(backup) == 0 || len(volname) == 0 || len(namespace) == 0 || len(scheduleName) == 0 {
+		return nil, CodedError(405, "failed to delete backup: Insufficient info -- required values volume_name, backup_name, namespace, schedule_name")
+	}
+
+	klog.Infof("deleting backup %v voluname %v namesace %v", backup, volname, namespace)
```

Comment:
lets print scheduleName also and correction in spelling is required

Reply:
ok